auth-web/src/components/auth/BCSCPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <v-container id="bcsc-container" class="my-10">
+    <h2 class="mt-5">Create an Account to Incorporate</h2>
+    <p class="my-5">Log in securely using your mobile BC Services Card, government's trust way to access online services</p>
+    <v-row>
+      <!-- Image Column -->
+      <v-col cols="12" md="5">
+        <v-img src="../../assets/img/BCSC-Helper.png" aspect-ratio="1.2" contain></v-img>
+      </v-col>
+      <v-spacer></v-spacer>
+      <!-- Info Column -->
+      <v-col cols="12" md="6" class="bcsc-info-col">
+        <h2>It's Secure</h2>
+        <v-list-item class="list-item" v-for="(item, index) in secureBulletPoints" :key="index" >
+          <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <h2>It's Quick and Easy</h2>
+        <v-list-item class="list-item" v-for="(item, index) in easeBulletPoints" :key="index" >
+          <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Panel Btns -->
+        <template v-if="!userProfile">
+          <v-btn large color="#fcba19" @click="emitLogin()" class="mt-5">
+            Log in with BC Services Card
+          </v-btn>
+          <p class="my-5">New to BC Registries? <a @click="emitAccountDialog()" class="create-account-link">
+            <u>Create a BC Registries Account</u></a>
```

Comment:
Does anchor text still need to be underlined explicitly? I thought that was a default browser style (and as such, override-able by user styling).

Reply:
This was a good question! Your right, by default it is underlined and it took me a few mins to find why this wasn't, which was due to the lack of a specified url, `href`.

Reply:
Great. For others: my concern here that hard-coding this style does not allow the user agent CSS to override it.